docs/references/javascript/clerk/handle-navigation.mdx

```diff
@@ -42,76 +38,29 @@ Take a look at the function parameters description below for more usage details.
 
 ### Props
 
-
-<Tables
-  headings={["Name", "Type", "Description"]}
-  headingsMeta={[{maxWidth:'300px'},{maxWidth: '300px'},{maxWidth: '300px'}]}
-  rows={[
-  {
-  cells: [
-  <code>params</code>,
-  <code><a href="#handle-magic-link-verification-params">HandleMagicLinkVerificationParams</a></code>,
-  <>Allows you to define the URLs where the user should be redirected to on successful verification and:
-  <ul><li>Completed sign in or sign up attempt.</li>
-  <li>Pending sign in or sign up attempt.</li>
+| Name | Type | Description |
+| --- | --- | --- |
+| `params` | [`HandleMagicLinkVerificationParams`](#handlemagiclinkverificationparams) | Allows you to define the URLs where the user should be redirected to on successful verification and:
+  <ul>
+    <li>Completed sign in or sign up attempt.</li>
+    <li>Pending sign in or sign up attempt.</li>
   </ul>
-  If the magic link is successfully verified on another device, there's a callback function parameter that allows custom code execution.</>,
-  ],
-  }
-    ,
-    {
-    cells: [
-    <code>customNavigate</code>,
-    <code>(to: string) {"=>"} Promise{"<"}unknown{">"}</code>,
-    <>Allows you to define a custom navigation function</>,
-    ],
-    }
-  ]}
-/>
+  If the magic link is successfully verified on another device, there's a callback function parameter that allows custom code execution. |
```

Comment:
Do multiline cells work like this? Amazing if so

Reply:
yup :~)

styling needs some work though. text in tables have weird line height anyways, and then li's have normal line-height i'm assuming.

<img width="1512" alt="Screenshot 2023-08-17 at 11 16 57" src="https://github.com/clerkinc/clerk-docs/assets/98043211/bb4c3ce4-5bae-4f09-aef9-acff1af16afb">